compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TaintAnalyzer.java

```diff
@@ -1826,19 +1779,17 @@ private void analyzeInvocation(BLangInvocation invocationExpr) {
                             break;
                         }
                     }
-                    analyzeArgument(paramIndex, invokableSymbol, invocationExpr, argExpr, returnTaintedStatus);
-                    if (stopAnalysis) {
-                        break;
+                    if (analyzeArgument(paramIndex, invokableSymbol, invocationExpr, argExpr)) {
```

Comment:
defaultableParamsInitialIndex which is used to calculate paramIndex is redundant